src/App/App.js

```diff
@@ -1,25 +1,40 @@
 import './App.css';
-import movieData from "../data";
 import Movies from '../movies/movies';
 import Header from '../header/header';
 import DetailPage from '../detailPage/detailPage';
-import { useState } from "react";
+import { useState, useEffect } from "react";
 
 function App() {
-  const [movieClicked, setMovieClicked] = useState(false)
-  const [singleMovie, setSingleMovie] = useState([])
+  const [movieClicked, setMovieClicked] = useState(false);
+  const [singleMovie, setSingleMovie] = useState([]);
+  const [movieData, setMovieData] = useState([]);
+  const [error, setError] = useState("");
+
+  const getMovies = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
Consider moving fetch calls to an apiCalls file - this helps promote separation of concerns!